curl/curl.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: curl
+spec:
+  params:
+   - name: url
+     description: URL to curl'ed
+     type: string
+   - name: options
+     description: options of url
+     type: array
+     default: []
+  steps:
+    - name: curl
+      image: curlimages/curl
```

Comment:
We should make this a param that defaults to `curlimages/curl`. This way users can pin to versions if needed.

Reply:
Thanks for the advice, I've changed my curl.yaml file and updated readme accordingly.